app/src/features/FhirResourceTree/TreeItem.tsx

```diff
@@ -44,33 +129,64 @@ const TreeItem = ({ elementNode }: TreeItemProps): JSX.Element => {
         break;
     }
   } else {
-    iconName = IconNames.LAYERS;
+    iconName =
+      elementNode.name === "extension"
```

Comment:
I know in this case `elementNode.name = elementNode.type` but it would be more correct to do this instead

```suggestion
      elementNode.type === "extension"
```